models/request.js

```diff
@@ -0,0 +1,74 @@
+var User = require('./user');
+var Service = require('./service');
```

Comment:
Don't need these since we're only concerned with `Request` associations in this file.